test/everypolitician_test.rb

```diff
@@ -195,4 +195,11 @@ def test_expose_the_statement_count_of_a_legislature
     end
   end
 
+  def test_lastmod_is_a_time
+    VCR.use_cassette('countries_json') do
+      australian_senate = Everypolitician.country(slug: 'Australia').legislature(slug: 'Senate')
+      assert_equal '2016-07-24 17:55:25 UTC', australian_senate.lastmod.to_s
```

Comment:
I don't think that's a sufficient test — this would still pass if `lastmod` simply returned a string.  You need to either keep the test you had before that it's a Time object, or if you don't want to bind it tightly to that, show that you can call some other methods on it than `to_s`.